test/countNumberOfLowercase.test.js

```diff
@@ -0,0 +1,27 @@
+import { equal } from 'assert';
+import { typeOfArgumentErrorMessage, countNumberOfLowercase } from '../countNumberOfLowercase.js';
+
+it('Mix of lowercase letters, uppercase letters, numbers and symbols in a string', () => {
```

Comment:
Do you think test cases like "lowercase at beginning and end" and "uppercase..." would improve test coverage (not talking about the mocha coverage, more like depth of your testing). If yes, could you please add them? If no, why?

Reply:
accepted, added new tests